backend/src/appointment/controller/apis/google_client.py

```diff
@@ -141,19 +141,27 @@ def list_events(self, calendar_id, time_min, time_max, token):
 
         return items
 
-    def create_event(self, calendar_id, body, token):
+    def save_event(self, calendar_id, body, token):
         response = None
         with build('calendar', 'v3', credentials=token, cache_discovery=False) as service:
             try:
                 response = service.events().import_(calendarId=calendar_id, body=body).execute()
             except HttpError as e:
-                logging.warning(f'[google_client.create_event] Request Error: {e.status_code}/{e.error_details}')
+                logging.warning(f'[google_client.save_event] Request Error: {e.status_code}/{e.error_details}')
                 raise EventNotCreatedException()
 
         return response
 
     def delete_event(self, calendar_id, event_id, token):
-        pass
+        response = None
+        with build('calendar', 'v3', credentials=token, cache_discovery=False) as service:
+            try:
+                response = service.events().delete(calendarId=calendar_id, eventId=event_id).execute()
```

Comment:
For some reason, this throws:

> [google_client.delete_event] Request Error: 404/[{'domain': 'global', 'reason': 'notFound', 'message': 'Not Found'}

Still investigating.

Reply:
That's very odd. We have the permissions to do so, so it shouldn't be that.

Reply:
Will do!

Reply:
Tested, works now 🎉